Products/ZenModel/UserInterfaceSettings.py

```diff
@@ -34,6 +34,7 @@ class UserInterfaceSettings(ZenModelRM):
         {'id': 'showEventSeverityIcons', 'type': 'boolean', 'mode': 'w'},
         {'id': 'enableTreeFilters', 'type': 'boolean', 'mode': 'w'},
         {'id': 'showPageStatistics', 'type' : 'boolean', 'mode':'w'},
+        {'id': 'renderNetworksFromCatalog', 'type' : 'boolean', 'mode':'w'},
```

Comment:
Sorry to invade the thread but on 4.2.5 sp 273 (plus hotfixes) the network page is unusably slow.  I read that there are performance impeocements in the latest rps havent tried that yet.

Actually would be great for us if those searches where permissioned.  We use ip realms but cant restrict access per realm or subnet now so i cant give customers access to the networks but up until now since it was slow..

the maps arent slow so I assumed it was using catalog searches for that so i again assumed the problem was that the right side is using catalog but the tree structure on the left wasnt and thats why it was slow..